package.json

```diff
@@ -45,5 +45,8 @@
         "uuid": "^9.0.0",
         "vite": "^4.5.5"
     },
-    "type": "module"
+    "type": "module",
+    "dependencies": {
+        "@creit.tech/stellar-wallets-kit": "^1.2.1"
```

Comment:
Will this work in the `devDependencies`? If so, that's probably preferable?

Reply:
I tried adding it in dev-dependecy but I got errors so i added it back to normal dependency